llvm/lib/Transforms/Instrumentation/NumericalStabilitySanitizer.cpp

```diff
@@ -850,7 +850,7 @@ void NumericalStabilitySanitizer::populateShadowStack(
   IRBuilder<> Builder(&CI);
   SmallVector<Value *, 8> ArgShadows;
   const bool ShouldCheckArgs = shouldCheckArgs(CI, TLI, CheckFunctionsFilter);
-  for (auto [ArgIdx, Arg] : enumerate(CI.operands())) {
+  for (auto &[ArgIdx, Arg] : enumerate(CI.operands())) {
```

Comment:
There is no copy of `Arg`. `Arg` is by reference, even if we don't use `auto &`

Reply:
Yes, you are correct. That is unnecessary and wrong, I have removed that.